app/models/item.rb

```diff
@@ -104,8 +105,21 @@ class Item < ApplicationRecord
       .or(where("base_items.category = 'Miscellaneous'"))
   }
 
+  before_create :set_reporting_category
   before_destroy :validate_destroy, prepend: true
 
+  # TODO: The enum names below have _enum appended in order to not clash with
+  # the scopes above. Remove _enum when the scopes above can be safely removed.
+  enum :reporting_category, {
+    adult_incontinence_enum: 0,
```

Comment:
Should be straightforward to remove the "_enum"'s appended here. Because the db is storing the integer values, so we won't need to do another db migration, just change the names here.

Reply:
You can still store string values - you can just make them the underscores instead of the full display names.

If scopes are an issue, you can just set `scopes: false`to not create the scopes. We can turn them back on later.

Reply:
Thanks, didn't know not creating those methods was an option. fixed in my last commit [420fb98](https://github.com/rubyforgood/human-essentials/pull/5034/commits/420fb9845e51edacf9fa6b9b6c183bbd274d1e81)